Libs/Mesh/CMakeLists.txt

```diff
@@ -161,3 +161,7 @@ INSTALL(TARGETS SurfaceToSurfaceDistance RUNTIME DESTINATION bin)
 #ADD_EXECUTABLE(fea2vtk fea2vtk.cxx)
 #TARGET_LINK_LIBRARIES(fea2vtk ${ITK_LIBRARIES} Mesh tinyxml)
 #INSTALL(TARGETS fea2vtk RUNTIME DESTINATION bin)
+
+ADD_EXECUTABLE(WarpMeshUsingLandmarks WarpMeshUsingLandmarks.cxx )
+TARGET_LINK_LIBRARIES(WarpMeshUsingLandmarks ${ITK_LIBRARIES} Mesh tinyxml Eigen3::Eigen igl::core)
+INSTALL(TARGETS WarpMeshUsingLandmarks RUNTIME DESTINATION bin)
```

Comment:
We can remove the standalone executable if the consolidated version exists.

Reply:
A consolidated version does not exist.  @riddhishb @sheryjoe @cchriste , what is the plan here?  Are we adding an executable and/or command for this?